Formula/elixir-ls.rb

```diff
@@ -0,0 +1,26 @@
+class ElixirLs < Formula
+  desc "Language Server and Debugger for Elixir"
+  homepage "https://elixir-lsp.github.io/elixir-ls"
+  url "https://github.com/elixir-lsp/elixir-ls/archive/refs/tags/v0.9.0.tar.gz"
+  sha256 "eaa2f5fd4ab9e497889e25419e4409cfaf808daca06259718668a71de8616e3f"
+  license "Apache-2.0"
+
+  depends_on "elixir"
+
+  def install
+    ENV["MIX_ENV"] = "prod"
+
+    system "mix", "local.hex", "--force"
+    system "mix", "local.rebar", "--force"
+    system "mix", "deps.get"
+    system "mix", "compile"
+    system "mix", "elixir_ls.release", "-o", prefix/"release"
+
+    libexec.install Dir["#{prefix}/release/*"]
+    bin.install Dir["#{libexec}/*"]
+  end
+
+  test do
+    assert_predicate bin/"launch.sh", :exist?
```

Comment:
We can't just check if the file exists. We should actually try and run it if it works, currently it can still be broken.

Reply:
Is the binary just called `launch.sh`? That's woefully generic. We should rename that.

Reply:
Ah I can replace it with the other entry point. The other `language_server.sh` just calls it internally directly.